templates/.circleci/config.yml.tpl

```diff
@@ -27,6 +27,20 @@ contexts: &contexts
 {{ $userContexts | toYaml | indent 2 }}
   ###EndBlock(extraContexts)
 
+# Branches used for releasing code, pre-release or not
+release_branches: &release_branches
+  {{- if $prereleases }}
+  # Release branch
+  - release
+  # Pre-releases branch
+  - {{ stencil.Arg "releaseOptions.prereleasesBranch" | default  .Git.DefaultBranch | squote }}
+    {{- if not (stencil.Arg "releaseOptions.prereleasesBranch") }}
```

Comment:
Should this `if` only go around the comment? Line 36 should take care of the entry for `.Git.DefaultBranch`. This looks like it will result in two entries with `.Git.DefaultBranch` if `prereleasesBranch` isn't there.